tests/functional/notifier.py

```diff
@@ -13,15 +13,18 @@
 GITHUB_BASE_URL = "https://api.github.com/repos"
 endpoint_data = {}
 
-def _set_endpoint_key( key, env_var ):
-   if key not in endpoint_data:
+
+def _set_endpoint_key(key, env_var):
+    if key not in endpoint_data:
```

Comment:
It still is because `_set_endpoint()` is called before `_verify_endpoint()`

Reply:
oh I see, I'm only reading from env variables and raise exception if no env variable is present.

previously it was setting default to `openshift-helm-charts/charts`, but it didn't seem to override with environment variables because `_verify_endpoint` was called before `_set_endpoint_key`..